### PR TITLE
Make etcd volume dependent on master instance

### PR DESCRIPTION
**Pull request by rossf7**
Cloud Formation template change so the etcd volume is deleted correctly.  Adds a DependsOn attribute for the EtcdVolume to the MasterInstance. Without this I've seen the guest stack fail to delete on `gauss`.

Previous error

screen shot 2018-02-22 at 3 19 02 pm

### CODE REVIEW

**Comment by rossf7** [comment]
Tested successfully locally.

**Comment by rossf7** [comment]
@fgimenez Ack. Running e2e now :)

File: service/awsconfig/v5/templates/cloudformation/guest/instance.go [hunk 1]
```diff
@@ -18,6 +18,8 @@ const Instance = `{{define "instance"}}
         Value: {{ .ClusterID }}-master
   EtcdVolume:
     Type: AWS::EC2::Volume
+    DependsOn:
+      - MasterInstance
```
Comment (xh3b4sd) [inline service/awsconfig/v5/templates/cloudformation/guest/instance.go]: I am not a YAML expert but I usually see lists without indentation. I think it should be this. 

```
+    DependsOn:
+    - MasterInstance
```

Comment (rossf7) [reply service/awsconfig/v5/templates/cloudformation/guest/instance.go]: Good point, done.

**Pull request merged**